debezium-core/src/main/java/io/debezium/relational/history/KafkaDatabaseHistory.java

```diff
@@ -155,14 +155,23 @@
             .withDescription("The unique identifier of the Debezium connector")
             .withNoValidation();
 
+    public static final Field KAFKA_QUERY_TIMEOUT_MS = Field.create(CONFIGURATION_FIELD_PREFIX_STRING + "kafka.query.timeout.ms")
+            .withDisplayName("Kafka admin client query timeout (ms)")
+            .withType(Type.LONG)
+            .withGroup(Field.createGroupEntry(Field.Group.ADVANCED, 2))
```

Comment:
@rk3rn3r, about having a new group `DATABASE_HISTORY` for this and all related properties?

Reply:
Sure, if you think a new group is needed you can introduce a new group for it any time.
pinging @mdrillin @indraraj @uidoyen

Reply:
As these are connection properties (for Kafka) and mandatory, they could also to the CONNECTION group. We do like that for the MySQL connector at the moment.

Reply:
@rk3rn3r this property is optional. Have default value of 3sec. Do you still think it belongs to CONNECTION?

Reply:
> We do like that for the MySQL connector at the moment.

@rk3rn3r, wasn't CONNECTION meant for the actual DB-connection related properties? And do you have a reference to where we put history-related properties into that group?

> Sure, if you think a new group is needed you can introduce a new group for it any time.

Ok, but what are the implications of this? Will this require a synced effort to get corresponding changes done to the UI? I.e. how exactly are these group definitions interpreted in the UI?

Reply:
@snigdhasjg @gunnarmorling Yes, this property is optional, but it belongs to the database history settings which are mandatory for connectors with database history. And all properties around database history are on the connection tab in the UI because they hold database history Kafka (broker) connection settings. So we decided this is a good place without introducing too many groups, especially as long as not all connectors support database history (yet).

@gunnarmorling 
> And do you have a reference to where we put history-related properties into that group?

For example we have it configured already here in the same file [KafkaDatabaseHistory.java#L100](https://github.com/debezium/debezium/blob/main/debezium-core/src/main/java/io/debezium/relational/history/KafkaDatabaseHistory.java#L100)

> what are the implications of this?

We should move all related properties to that new group and then we need to change the UI to show these properties accordingly on the connection tab (or wherever you want to place it). But this change could be done safely now as it will only be in effect when the UI migrates to the JSON schemas.

Does this help?

Reply:
> But this change could be done safely now as it will only be in effect when the UI migrates to the JSON schemas.

@rk3rn3r, are you sure about this? Isn't that grouping/ordering already obtained today also with the current approach of having the Debezium connectors embedded into the UI backend and dynamically building the JSON schema from that? Or are you saying it's safe, as we don't update that embedded version to 1.9-next (which would bring this change) just yet?

Reply:
@gunnarmorling 100% sure. We are using 1.9-SNAPSHOT as dependency and nightly images. But until we migrate the grouping comes from the big maps in the `ConnectorIntegrator` implementations. For example see [PostgresConnectorIntegrator](https://github.com/debezium/debezium-ui/blob/main/backend/src/main/java/io/debezium/configserver/service/postgres/PostgresConnectorIntegrator.java#L41-L127)

Reply:
@rk3rn3r Used group`CONNECTION` index 33

Not sure what else I'm missing here.

Reply:
That's good @snigdhasjg. Really appreciate your effort!
Last the final question to @gunnarmorling if he prefers a new group `DATABASE_HISTORY` now or if we leave it like this and create a new JIRA to cover new group creation?